v3/how-to-guides/01-basics/e-custom-chainspec/index.mdx

```diff
@@ -79,25 +87,25 @@ With a plain spec avalible, you generate a final raw chain spec by:
     "
 />
 
-It is _conventional_ to include the chain specification files for your node _within the source code itself_ so that anyone building your node and avoid any chance of non-determinism causing nodes to belive they are on different networks (as the genesis blob may differ).
-Polkadot, Kusama, Rococo, and more network chainpec files are found [in the source here](https://github.com/paritytech/polkadot/tree/master/node/service/res) along with a `.gitignore` file to ensure that you don't acidentally change these `!/*.json` files as you build futher on your node's software and do [runtime upgrades](/tutorials/v3/forkless-upgrades).
+It is _conventional_ to include the chain specification files for your node _within the source code itself_ so that anyone building your node and avoid any chance of non-determinism causing nodes to believe they are on different networks (as the genesis blob may differ).
```

Comment:
```suggestion
It is _conventional_ to include the chain specification files for your node _within the source code itself_ so that anyone can build your node in the same way, whereby it becomes easy to check for non-determinism by comparing a genesis blob with another. 
```
This could probably be simplified further..

Reply:
@lisa-parity probably has the trick.